### ISSUE TITLE
Openpose + face?

### ISSUE THREAD

**Issue by mercantigo** [williamyang1991/Rerender_A_Video#52]
Hi
Is there any way to use Openpose or dwpose with face support + canny?
I think that output results could be greatly improved with this

**Comment by williamyang1991** [comment]
Our method is compatible to different controlnet models. 
So you can customize your controlnet choice.
To add other controlnet, you may refer to the `Using customized models` section of [webui](https://github.com/williamyang1991/Rerender_A_Video#webui-recommended)

And to use openpose controlnet and canny controlnet simultanously, 
you can add the multiple control features to the SD feature rather than a single control feature, (you need to modify the code)
https://github.com/lllyasviel/ControlNet/blob/ed85cd1e25a5ed592f7d8178495b4483de0331bf/cldm/cldm.py#L35
`h += control.pop()` to `h += control1.pop() + control2.pop()`

https://github.com/lllyasviel/ControlNet/blob/ed85cd1e25a5ed592f7d8178495b4483de0331bf/cldm/cldm.py#L41
`h = torch.cat([h, hs.pop() + control.pop()], dim=1)` to `h = torch.cat([h, hs.pop() + control1.pop() + control2.pop()], dim=1)`
